e2eshark/onnx/models/opt-125M-awq/model.py

```diff
@@ -16,34 +16,92 @@
 # to form runmodel.py in the rundirectory which is then taken
 # through flow
 
-# set some variable parameters (dynamic dims)
-batch_size = 3 
-sequence_length = 7 
-past_sequence_length = 0 # setting this parameter to anything other than 0 results in an error
+# If you want to run the test with a static model.onnx, set run_as_static = True
+run_as_static = False
+dim_params = (
+    "batch_size",
+    "sequence_length",
+    "past_sequence_length",
+    "past_sequence_length + 1",
+)
+dim_values = (12, 7, 0, 1)
+pv_zip = zip(dim_params, dim_values)
+pv = dict(pv_zip)
+
+model = onnx.load("model.onnx")
+isDynamic = bool(model.graph.input[0].type.tensor_type.shape.dim[0].dim_param)
+
+if run_as_static and isDynamic:
+    print("Saving dynamic model...")
+    onnx.save(model, "dynamic_model.onnx")
+    print("\t dynamic model saved as dynamic_model.onnx")
```

Comment:
May want to use `logging` (https://docs.python.org/3/library/logging.html) with a scoped logger if adding so many `print` calls. That way developers could redirect logging, change which modules log at what verbosity, and see which logs come from which file.